toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -631,11 +656,11 @@ func configureSystemFiles(installChroot *safechroot.Chroot, hostname string, con
 	return
 }
 
-func calculateTotalPackages(packages []string, installRoot string) (totalPackages int, err error) {
+func calculateTotalPackages(packages []string, installRoot string) (installedPackages *repocloner.RepoContents, err error) {
```

Comment:
nit: as an effort to improve readability of toolkit, if you could add a comment to the functions you touch upon elucidating their intended usecase and input arguments, that would be very helpful

Reply:
Sure, will update